Task: remove lerna

esclapes opened the pull request:
### Purpose

Bumping all packages and removing lerna details

### Notes

### Review

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.account.buffer.com :smile:_

hharnisc commented:
@kiriappeee wanted to put this build on your radar. Looks like kubedashed failed for some reason: https://jenkins.k8s.bufferapp.com/job/Build-Deployment/6386/console

hharnisc commented:
This is right on track @esclapes great work getting things started here and it is looking very close :raised_hands: 

After taking a quick look at the tests, it seems there's a couple minor linter errors:

https://travis-ci.org/bufferapp/buffer-account/builds/449774811#L498

esclapes commented:
Hey @hharnisc thanks for the heads up about the lining issues. I have fixed them, and also updated a couple of snapshot tests that were also failing.

I guess this was already broken on `next`; it makes sense to bundle the fix here.

Please, let me know if you spot anything else.

hharnisc commented:
Thanks for doing the much needed linter and snapshot test updates @esclapes the code is looking great :ok_hand: 

Hey @kiriappeee if you've got some time today could you take a look at the deployment: https://jenkins.k8s.bufferapp.com/job/Build-Deployment/6406/console

Not sure exactly why this one is failing but it leaves the github status in a failed state for bufferbotbrains/cicd :thinking:

hharnisc commented:
@esclapes once the deploy passes I think this is good to go :+1: I'll give this a pending :shipit:

kiriappeee commented:
@hharnisc feel free to ignore that. That's a temporary issue which will go away once everything is moved over to the multi yaml format. Apologies that that exists. I'll try to make it go away 😁
